systemd/ceph-disk@.service

```diff
@@ -1,5 +1,7 @@
 [Unit]
 Description=Ceph disk activation: %f
+After=local-fs.target
+Wants=local-fs.target
```

Comment:
It would make sense to me also. I chose to go with the same kind of dependency as the other .service and .target defined by Ceph. As you say: if that is not present...we're in trouble big time ;-)